web-api-db-ctrl/postgres/tenant/20230712141144_document.sql

```diff
@@ -0,0 +1,31 @@
+-- Copyright 2023 Xayn AG
+--
+-- This program is free software: you can redistribute it and/or modify
+-- it under the terms of the GNU Affero General Public License as
+-- published by the Free Software Foundation, version 3.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU Affero General Public License for more details.
+--
+-- You should have received a copy of the GNU Affero General Public License
+-- along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+CREATE TYPE preprocessing_step AS ENUM (
+    'none',
+    'split',
+    'summarize'
+);
+
+ALTER TABLE document
+    ALTER COLUMN is_summarized DROP DEFAULT,
+    ALTER COLUMN is_summarized TYPE preprocessing_step
+        USING CASE
+            WHEN is_summarized THEN 'summarize'::preprocessing_step
+            ELSE 'none'::preprocessing_step
+        END,
+    ALTER COLUMN is_summarized SET DEFAULT 'none';
+
+ALTER TABLE document
+    RENAME COLUMN is_summarized TO preprocessing_step;
```

Comment:
no they are different name spaces, you can e.g. name a column  pretty much anything as long as it's not a keyword, through if there is a bad overlap with e.g. aggregation functions depending on SQL syntax it can be confusion for a human